Dockerfile

```diff
@@ -26,9 +26,9 @@ RUN pip3 config set global.index-url https://pypi.tuna.tsinghua.edu.cn/simple &&
 
 # https://forum.aspose.com/t/aspose-slides-for-net-no-usable-version-of-libssl-found-with-linux-server/271344/13
 # aspose-slides on linux/arm64 is unavailable
-RUN --mount=type=bind,source=libssl1.1_1.1.1f-1ubuntu2_amd64.deb,target=/root/libssl1.1_1.1.1f-1ubuntu2_amd64.deb \
-    if [ "${ARCH}" = "amd64" ]; then \
-        dpkg -i /root/libssl1.1_1.1.1f-1ubuntu2_amd64.deb; \
+RUN --mount=type=bind,source=libssl1.1_1.1.1f-1ubuntu2_${ARCH}.deb,target=/root/libssl1.1_1.1.1f-1ubuntu2_${ARCH}.deb \
```

Comment:
Let download_deps.py to prepare libssl*.deb if necessary.
I can build image on an Google Cloud Ubuntu 24.04 linux/arm64 without  libssl1.1_1.1.1f-1ubuntu2_arm64.deb. 
So this deb package is optional for linux/arm64.